files/waydroid-full-ui.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
I love the idea of Android integration! Do we want to use Anbox or Waydroid? Do you know which is better? I'd prefer to only have one solution for it.

Reply:
Id Recommend waydroid when using applications there will be less lag

Reply:
i have fully removed Anbox since i was experiencing an FPS downgrade while testing it on Games